tests/request.test.js

```diff
@@ -34,6 +34,7 @@ describe('requests', () => {
 
       const json = await get('api/pokemon/1');
       expect(json).toEqual({ id: 1 });
+      expect(middleware.parseJSON).toHaveBeenCalledWith(expect.any(Promise), expect.any(String), undefined);
```

Comment:
Why the any string? It should be the `/api/pokemon/1` url right?

Reply:
Yeah you're right, I'll change the test.